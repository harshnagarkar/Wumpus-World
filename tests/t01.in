ab-cdefgh
-abcdefgh
id
3000
