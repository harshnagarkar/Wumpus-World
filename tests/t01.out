 a b -
 c d e
 f g h

 a - b
 c d e
 f g h

 - a b
 c d e
 f g h

Total nodes: 6, max depth is: 2
